MapboxDirections/MBRouteOptions.swift

```diff
@@ -211,6 +211,10 @@ open class RouteOptions: NSObject {
      */
     open var routeShapeResolution = RouteShapeResolution.low
     
+    open var segmentAttributes: [Attribute] = []
+    
+    open var nodeAttributes: [Attribute] = []
```

Comment:
Compared to the situation in RouteLeg before, there’s no need to distinguish between segment and node attributes in RouteOptions – the two behave exactly the same as far as requesting a route is concerned. A single `attributeOptions` property will do.